scripts/e2e/spm.sh

```diff
@@ -8,13 +8,13 @@ set -euf -o pipefail
 print_help() {
   echo "Usage: $0 [-b binary] [-d database]"
   echo "-b: Which binary to build: 'all-in-one' or 'jaeger' (v2, default)"
-  echo "-d: Which database to use: 'memory' (default) or 'elasticsearch'"
+  echo "-d: Which database to use: 'prometheus' (default) or 'elasticsearch'"
```

Comment:
I am not sure I follow. You still need to distinguish between memory vs. ES for trace storage, where `prometheus` is never a valid value. Are you simply inferring that trace store should be `memory` if metrics store is `prometheus`?
```suggestion
  echo "-d: Which database to use as metrics store: 'prometheus' (default) or 'elasticsearch'"
```

Reply:
Hi @yurishkuro, sorry for this misunderstanding, I plan to change this database to metricstore. So I replace memory with Prometheus for this new implementation